h2o-r/tests/testdir_algos/glm/runit_PUBDEV_8947_GLM_likelihoods_aic.R

```diff
@@ -0,0 +1,77 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../../scripts/h2o-r-test-setup.R")
+
+test.glm.aic.likelihood <- function() {
+  # checking for Gaussian
+  h2o.data = h2o.uploadFile(locate("smalldata/prostate/prostate_complete.csv.zip"), destination_frame="h2o.data")    
+  R.data = as.data.frame(as.matrix(h2o.data))
+  myY = "GLEASON"
+  myX = c("ID","AGE","RACE","CAPSULE","DCAPS","PSA","VOL","DPROS")
+  R.formula = (R.data[,"GLEASON"]~.) 
+  model.h2o.gaussian.identity <- h2o.glm(calc_like=TRUE, x=myX, y=myY, training_frame=h2o.data, family="gaussian", link="identity",alpha=0.5, lambda=0, nfolds=0)
+  model.R.gaussian.identity <- glm(formula=R.formula, data=R.data[,2:9], family=gaussian(link=identity), na.action=na.omit)
+  perf <- h2o.performance(model.h2o.gaussian.identity, h2o.data)
```

Comment:
change 

perf <- h2o.performance(model.h2o.gaussian.identy, h2o.data)

to

perf <- h2o.performance(model.h2o.gaussian.identy, train=TRUE)